prep-css-flexbox/align-items.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Flexbox - align-items</title>
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="starwars.css">
+  <style>
+    /* your styles here 👇 */
+    #card-deck {
+      display: flex;
+      flex-basis: 33%;
```

Comment:
`flex-basis` can only be applied to "flex items", that is the children of an element with `display: flex;` applied to it, so you will need a separate rule set with the selector for `.card-wrapper`.

Reply:
Updated
![Screenshot updated CSS Flex Box Exercise - Revision](https://user-images.githubusercontent.com/97073517/149644809-7fd01452-e9b6-47c0-83f8-9e76be3f9f10.png)